cloud/aws/locals.tf

```
@@ -1,8 +1,7 @@
 locals {
   integration_name = "AWSIntegration${var.suffix == "" ? "" : "-${title(var.suffix)}"}"
-  open_aws_services = [
+  open_aws_services = coalescelist(var.included_services, [
     for service in data.signalfx_aws_services.aws_services.services[*].name :
     service if !contains(concat(["AWS/EC2"], var.excluded_services), service)
-  ]
+  ])
```

I'd rather alter the logic to ensure only known services are included, `AWS/EC2` is never included, and not ignore `excluded_services` variable:

```suggestion
  open_aws_services = [
    for service in data.signalfx_aws_services.aws_services.services[*].name :
    service if contains(var.included_services, service) && !contains(concat(["AWS/EC2"], var.excluded_services), service)
  ]
```

---

I implemented the behavior you described in a different way, let me know what you think but:
* only known services are included
* excluded services is used if you specify your list or if you include every services
* AWS/EC2 is never included as it has its special configuration